### ISSUE TITLE
Error list refers to "Code Analysis results window"

### ISSUE THREAD

**Issue by bkoelman** [dotnet/roslyn#4254]
In VS2015 RTM, I tried building a 2013 solution which contains a custom ruleset that includes an FxCop extension. This displays in the Error List:

```
Code Analysis detected errors.  See Code Analysis results window or log file for details.
```

![ca-window](https://cloud.githubusercontent.com/assets/10324372/9005164/2195050e-377c-11e5-8444-ecceeadac122.png)

However, I cannot find a window named "Code Analysis results" in VS2015. I believe it would be more helpful to instead point to the Build Output window, which contains:

```
1>  Running Code Analysis...
1>MSBUILD : error : CA0053 : Unable to load rule assembly 'e:\bart\documents\visual studio 2015\projects\agility2015\agilityeventmanagementsystem\lib\codecontractnullabilityfxcoprules.dll': Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
1>MSBUILD : warning : CA0062 : The rule 'CH1000' referenced in rule set 'e:\Bart\Documents\Visual Studio 2015\Projects\Agility2015\AgilityEventManagementSystem\Circe\..\AgilityRuleSet.ruleset' could not be found.
1>MSBUILD : warning : CA0062 : The rule 'CH1001' referenced in rule set 'e:\Bart\Documents\Visual Studio 2015\Projects\Agility2015\AgilityEventManagementSystem\Circe\..\AgilityRuleSet.ruleset' could not be found.
1>MSBUILD : warning : CA0062 : The rule 'CH1002' referenced in rule set 'e:\Bart\Documents\Visual Studio 2015\Projects\Agility2015\AgilityEventManagementSystem\Circe\..\AgilityRuleSet.ruleset' could not be found.
1>MSBUILD : warning : CA0062 : The rule 'CH1003' referenced in rule set 'e:\Bart\Documents\Visual Studio 2015\Projects\Agility2015\AgilityEventManagementSystem\Circe\..\AgilityRuleSet.ruleset' could not be found.
1>  Code Analysis Complete -- 1 error(s), 4 warning(s)
```

**Comment by srivatsn** [comment]
The codeanalysis window has been removed in VS 2015 and the issues that used to be reported there are now reported in the error list itself. So the warning from CA0062 etc that you see in the build output window should be part of the 24 warnings in the error list.

This error is now defunct and should just be removed entirely. Thanks for reporting this issue,.

**Comment by bkoelman** [comment]
I agree that's a better solution, thanks for explaining.

**Comment by mavasani** [comment]
Fixed with VSO commit https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/commit/b41d5818554b3bd09b02fbd2b825747f2a3ddc71

**Comment by mavasani** [comment]
@heejaechang can you please verify?
